django-stubs/conf/global_settings.pyi

```diff
@@ -189,7 +192,7 @@ STATIC_ROOT: Optional[str] = ...
 STATIC_URL: Optional[str] = ...
```

Comment:
surprised it didn't update Optional[str] to str | None

Reply:
Yeah this will be updated in the next PR I prepared to use PEP-604 (`A | B` and `A | None`) syntax. The changes are huge I had to split them to multiple PRs.